libs/kv/types.proto

```diff
@@ -17,6 +17,6 @@ option (gogoproto.testgen_all)  = true;
 // Abstract types
 
 message Pair {
-  bytes key   = 1;
```

Comment:
is this due to you running `make proto-lint`?

Reply:
Yes, it is.